tests/x11/chromium.pm

```diff
@@ -12,41 +12,29 @@ use Mojo::Base 'x11test', -signatures;
 use testapi;
 use utils;
 
-# Prevent lost characters due to temporary unresponsiveness of chromium address bar.
-# It looks like chromium becomes unresponsive after typing about 3
-# characters, likely due to some auto-completion feature.
+# Avoid lost characters due to temporary unresponsiveness by re-launching
+# with the correct URL via the command-line.
 # See https://progress.opensuse.org/issues/109737 for details
-sub type_address ($string) {
-    type_string substr($string, 0, 10), @_, max_interval => utils::SLOW_TYPING_SPEED;
-    enter_cmd substr($string, 10), @_;
+sub launch_with_url ($url) {
+    # avoid async keyring popups
+    x11_start_program('chromium --password-store=basic ' . $url, target_match => 'chromium-main-window', match_timeout => 50);
 }
 
 sub run {
     select_console 'x11';
     mouse_hide;
     ensure_installed 'chromium';
 
-    # avoid async keyring popups
-    x11_start_program('chromium --password-store=basic', target_match => 'chromium-main-window', match_timeout => 50);
-
-    wait_screen_change { send_key 'esc' };    # get rid of popup (or abort loading)
-    send_key 'ctrl-l';    # select text in address bar
-
-    # Additional waiting to prevent unready address bar
-    # https://progress.opensuse.org/issues/36304
-    assert_screen 'chromium-highlighted-urlbar';
-    type_address('chrome://version ');
+    launch_with_url('chrome://version');
     assert_screen 'chromium-about';
```

Comment:
Using `$target_match` within the `x11_start_program` and then `assert_screen` is redundant. Better pass on a parameter `$target_match` as well:

```suggestion
    launch_with_url('chrome://version', target_match => 'chromium-about');
```

I would likely use only named parameters and make it

```suggestion
    launch(url => 'chrome://version', target_match => 'chromium-about');
```

Reply:
Actually the url is the important part here, the rest we can pass through directly